src/Evolution/Systems/Ccz4/Ricci.hpp

```diff
@@ -0,0 +1,71 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace Ccz4 {
+/// @{
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes the spatial ricci tensor
+ *
+ * \details Computes the spatial ricci tensor as:
+ * \f{align}
+ *     R_{ij} &=
+ *       \partial_k \Gamma^m{}_{ij} - \partial_j \Gamma^m{}_{ik} +
+ *       \Gamma^l{}_{ij} \Gamma^m{}_{lk} - \Gamma^l{}_{ik} \Gamma^m{}_{lj}
+ * \f}
+ * where
+ * \f{align}
+ *     \partial_k \Gamma^m{}_{ij} &=
+ *       \partial_k \tilde{\Gamma}^m{}_{ij} +
+ *       2 D_k{}^{ml} (\tilde{\gamma}_{jl} P_i + \tilde{\gamma}_{il} P_j
+ *         - \tilde{\gamma}_{ij} P_l) -
+ *       2 \tilde{\gamma}^{ml} (D_{kjl} P_i + D_{kil} P_j - D_{kij} P_l) -
+ *       \tilde{\gamma}^{ml} (
+ *         \tilde{\gamma}_{jl} \partial_{(k} P_{i)} +
+ *         \tilde{\gamma}_{il} \partial_{(k} P_{j)} -
+ *         \tilde{\gamma}_{ij} \partial_{(k} P_{l)})
+ * \f}
+ * \f$\Gamma^k{}_{ij}\f$, \f$\partial_m \tilde{\Gamma}^k{}_{ij}\f$,
+ * \f$\tilde{\gamma}_{ij}\f$, \f$\tilde{\gamma}^{ij}\f$, \f$D_{ijk}\f$,
+ * \f$D_k{}^{ij}\f$, \f$P_i\f$, and \f$\partial_j P_{i}\f$ are the spatial
+ * christoffel symbols of the second kind defined by
+ * `Ccz4::Tags::ChristoffelSecondKind`, the spatial derivative of the conformal
+ * spatial christoffel symbols of the second kind defined by
+ * `Ccz4::Tags::DerivConformalChristoffelSecondKind`, the conformal spatial
+ * metric defined by `Ccz4::Tags::ConformalMetric`, the inverse conformal
+ * spatial metric defined by `Ccz4::Tags::InverseConformalMetric`, the CCZ4
+ * auxiliary variable defined by `Ccz4::Tags::FieldD`, the CCZ4 identity defined
+ * by `Ccz4::Tags::FieldDUp`, the CCZ4 auxiliary variable defined by
+ * `Ccz4::Tags::FieldP`, and its spatial derivative, respectively.
+ */
+template <size_t Dim, typename Frame, typename DataType>
+void ricci(
```

Comment:
Note that for some reason, while the documentation does properly generate for this function, its page does not properly get linked to the search result for "ricci".

Reply:
The `details` math in this documentation is identical to the `details` math in the documentation for the `Ccz4::Ricci` tag, but the link to the `Ccz4::Ricci` tag is properly linked to the search result. Perhaps something is wrong with the `\ingroup`? I've been using `\ingroup GeneralRelativityGroup` for all `Ccz4` functions, and they are properly linked on the website outside of this `ricci` one.